README.md

```diff
@@ -99,15 +99,39 @@ This documentation assumes that your build directory is a subdirectory of the
 be specified by setting ARGS before invoking ``do_cmake.sh``.  See [cmake
 options](#cmake-options) for more details. For example:
 
-    ARGS="-DCMAKE_C_COMPILER=gcc-7" ./do_cmake.sh
-
-To build only certain targets, run a command of the following form:
-
-	ninja [target name]
+       ninja -j3
+
+   > [IMPORTANT]
```

Comment:
Did you want to use the new [markdown note syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)?
```suggestion
   > [!IMPORTANT]
```

Reply:
I do. I'll try it out.